misc/python/materialize/cli/mzconduct.py

```diff
@@ -598,6 +601,19 @@ def run(self, comp: Composition, workflow: Workflow) -> None:
         print("Workflow has environment of", workflow.env)
 
 
+@Steps.register("sleep")
+class Sleep(WorkflowStep):
+    """Waits for the defined duration of time."""
+
+    def __init__(self, path: Path, duration: int) -> None:
+        super().__init__(path)
+        self._duration = duration
```

Comment:
I thought you were going to add validation here?

Reply:
I thought I had done that too! I'll add it in.

Reply:
Oh, wait, I decided against adding validation here! We've removed *almost all* of the reasons for using sleep and the last remaining sleep calls are not user configurable. They are always integers within the YAML file. I'm also planning to remove sleep, so I think it's best we leave this as is for now.